pypsa/linopf.py

```diff
@@ -720,6 +720,11 @@ def network_lopf(n, snapshots=None, solver_name="cbc",
     if formulation != "kirchhoff":
         raise NotImplementedError("Only the kirchhoff formulation is supported")
 
+    if n.generators.committable.any():
+        logger.warn("Unit commitment is not yet implemented for optimsation "
+           "without using pyomo. The following generators will be treated as "
+          f"non-commitables:\n{list(n.generators.query('committable').index)}")
```

Comment:
optimsation -> optim*i*sation
non-commitables -> non-commit*t*ables

Unit *commitment* is correct.

Reply:
yep